ee/modules/450-keepalived/hooks/keepalived_instance_handler_test.go

```diff
@@ -7,7 +7,6 @@ package hooks
 
 import (
 	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
```

Comment:
Is it OK? I see `Expect` calls from gomega in this file.

Reply:
It is a bug. Fixed. Thanks